services/bitso/api-handler.js

```diff
@@ -0,0 +1,21 @@
+const request = require('request-promise');
```

Comment:
`request-promise` is not in the `package.json` and I'd like to use the [native promise](https://github.com/request/request-promise-native) version.